CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Add support for update, partial_update, delete and get_all methods in fluent API bindings.
```

Comment:
Will it be better if we can make a release? Thus it can be operated alone

Reply:
There is no need to release it immediately. We still need to implement rest of the apis before we enable this feature.